dex/src/main/scala/com/wavesplatform/dex/SpendableBalancesActor.scala

```diff
@@ -0,0 +1,89 @@
+package com.wavesplatform.dex
+
+import akka.actor.{Actor, ActorRef}
+import akka.pattern.pipe
+import cats.instances.long.catsKernelStdGroupForLong
+import cats.syntax.group.catsSyntaxGroup
+import com.wavesplatform.dex.domain.account.Address
+import com.wavesplatform.dex.domain.asset.Asset
+import com.wavesplatform.dex.fp.MapImplicits.cleaningGroup
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class SpendableBalancesActor(spendableBalances: (Address, Set[Asset]) => Future[Map[Asset, Long]],
+                             allAssetsSpendableBalances: Address => Future[Map[Asset, Long]],
+                             addressDirectory: ActorRef)(implicit ec: ExecutionContext)
```

Comment:
Consider remove `ec`. `context.dispatcher` is fine here, because blocking `allAssetsSpendableBalances` and `spendableBalances` will be run with grpc context.

Reply:
Done